packages/fetchai/skills/buyer_aw2/README.md

```diff
@@ -0,0 +1,24 @@
+# Car Park Client
```

Comment:
# Buyer AW2

Reply:
Either change this to `simple buyer aw2` to the seller skill to `seller aw2` to make their names consistent.